src/cli/worker_manager.go

```diff
@@ -235,6 +235,11 @@ func populateResolverConfig(gc *CLIConf) *zdns.ResolverConfig {
 			log.Fatal("unable to parse blacklist file: ", err)
 		}
 	}
+	// This must occur after setting the DNSConfigFilePath above, so that ZDNS knows where to fetch the DNS Config
+	config, err := populateIPTransportMode(gc, config)
+	if err != nil {
+		log.Fatal("could not populate IP transport mode: ", err)
+	}
 	// This is used in extractAuthorities where we need to know whether to request A or AAAA records to continue iteration
 	// Must be set after populating IPTransportMode
```

Comment:
Noticed we said this had to be "after populating IPTransportMode" and set transport mode after 😢